src/EPG.cpp

```diff
@@ -132,13 +129,65 @@ PVR_ERROR EPG::GetEPGForChannel(int channelUid, time_t start, time_t end, kodi::
         }
 
       }
-      broadcast.SetSeriesNumber(XMLUtils::GetIntValue(pListingNode, "season", EPG_TAG_INVALID_SERIES_EPISODE));
-      broadcast.SetEpisodeNumber(XMLUtils::GetIntValue(pListingNode, "episode", EPG_TAG_INVALID_SERIES_EPISODE));
+
+      int season{ EPG_TAG_INVALID_SERIES_EPISODE };
```

Comment:
Remove space before and after the value?

Reply:
Done